app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.java

```diff
@@ -15,8 +15,6 @@
 public final class TextViewBindingAdapters {
 
   private static int MINUTE_MILLIS = (int) TimeUnit.MINUTES.toMillis(1);
```

Comment:
For consistency, I suggest inlining this as well.

Reply:
Done